src/main/java/com/exactpro/th2/mstore/AbstractMessageStore.java

```diff
@@ -316,11 +330,11 @@ private static void verifySequence(int messageIndex, long lastSeq, long currentS
         }
     }
 
-    private static void verifySession(int messageIndex, SessionKey lastKey, SessionKey sessionKey) {
-        if (!lastKey.equals(sessionKey)) {
+    private static void verifySessionGroup(int messageIndex, SessionKey lastKey, SessionKey sessionKey) {
```

Comment:
it makes sense to discard this function and move it's logic in caller function

Reply:
Ok